Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -187,6 +187,11 @@ public override IEnumerable<Guid> ListChainIds()
                 {
                     continue;
                 }
+                catch (Exception e)
+                {
+                    LogUnexpectedException(nameof(ListChainIds), e);
+                    continue;
+                }
```

Comment:
Strictly speaking, this change changes the semantics so that unexpected exceptions are not thrown out of this block. In general, these changes don't cause much of a problem, but if you don't intend to change the behavior, it's better to use `throw`.

```suggestion
                catch (Exception e)
                {
                    LogUnexpectedException(nameof(ListChainIds), e);
                    throw;
                }
```